docs/examples/network-policies/allow-inter-node-traffic.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-inter-node-traffic
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: rabbitmq
```

Comment:
The same suggestion applies for the other network policies as well. Do you think we should mention that people could set labels to only matches a particular RabbitmqCluster. The label `app.kubernetes.io/name` is set to the name of the CR.